diff -ruN orig/scripts/ssr-glinject new/scripts/ssr-glinject
--- a/scripts/ssr-glinject	2019-01-22 17:29:57.000000000 -0600
+++ scripts/ssr-glinject	2019-02-03 19:21:17.353769171 -0600
@@ -59,6 +59,6 @@
 	fi
 done
 
-echo "ssr-glinject: LD_PRELOAD = $LD_PRELOAD:libssr-glinject.so"
+echo "ssr-glinject: LD_PRELOAD = $LD_PRELOAD:/usr/\$LIB/simplescreenrecorder/libssr-glinject.so"
 echo "ssr-glinject: command = $@"
-LD_PRELOAD="$LD_PRELOAD:libssr-glinject.so" "$@"
+LD_PRELOAD="$LD_PRELOAD:/usr/\$LIB/simplescreenrecorder/libssr-glinject.so" "$@"
diff -ruN orig/src/AV/Input/GLInjectInput.cpp new/src/AV/Input/GLInjectInput.cpp
--- a/src/AV/Input/GLInjectInput.cpp	2019-01-22 17:29:57.000000000 -0600
+++ src/AV/Input/GLInjectInput.cpp	2019-02-03 19:23:12.349929589 -0600
@@ -97,7 +97,7 @@
 bool GLInjectInput::LaunchApplication(const QString& channel, bool relax_permissions, const QString& command, const QString& working_directory) {
 
 	// prepare command
-	QString full_command = "LD_PRELOAD=\"libssr-glinject.so\" ";
+	QString full_command = "LD_PRELOAD=\"/usr/\\$LIB/simplescreenrecorder/libssr-glinject.so\" ";
 	full_command += "SSR_CHANNEL=\"" + ShellEscape(channel) + "\" ";
 	if(relax_permissions)
 		full_command += "SSR_STREAM_RELAX_PERMISSIONS=1 ";
